addons/mail/static/src/core/common/message.js

```diff
@@ -304,6 +308,17 @@ export class Message extends Component {
         });
     }
 
+    onClickDownloadAll() {
+        const files = this.props.message.attachments;
+        download({
+            data: {
+                file_ids: files.map(rec => rec.id),
```

Comment:
This `const` is useless: only used once, and LOC is not that complex where it's used